components/brave_vpn/browser/api/brave_vpn_api_request.cc

```diff
@@ -265,7 +265,7 @@ void BraveVpnAPIRequest::OnGetSubscriberCredential(
   bool success = api_request_result.response_code() == 200;
   std::string error;
   std::string subscriber_credential =
-      ParseSubscriberCredentialFromJson(api_request_result.body(), &error);
+      ParseSubscriberCredentialFromJson(api_request_result.TakeBody(), &error);
   if (!success) {
     subscriber_credential = error;
     VLOG(1) << __func__ << " Response from API was not HTTP 200 (Received "
```

Comment:
ParseSubscriberCredentialFromJson migth return just a single string

Reply:
It's not clear to me. Do you mean to remove `std::string* error`?
`error` as `subscriber_credential` is passed to the callback.